launcher/minecraft/PackProfile.h

```diff
@@ -48,6 +48,7 @@
 
 #include "Component.h"
 #include "LaunchProfile.h"
+#include "minecraft/PackProfile_p.h"
```

Comment:
Do we need this included?
I do not like the private header stuff but if we use them we should not include them inside the normal headers.
Is there a good reason why we should use them in favor of private members?

Reply:
it *was* needed for the `ComponentContainer` type alias but I can just specify the full type